models/articles.model.js

```diff
@@ -18,4 +18,17 @@ function selectComByArticle_id(id){
         ORDER BY comment_id DESC;
     `, [id]);
 } 
-module.exports = { selectArticleById, selectComByArticle_id }
\ No newline at end of file
+
+function insertComByArticle_id(article_id, username, body, article,){
+   
+        const { votes, created_at} = article
+
+        return db
+        .query(
+          'INSERT INTO comments (body, article_id, author,votes,created_at) VALUES ($1, $2, $3, $4, $5) RETURNING *;',
```

Comment:
I'd take a look at the tables here and see what is needed. votes and created_at are added by PSQL because of the way we have set it up so we don't need to add them ourselves.